arc/src/arc_aio.c

```diff
@@ -32,7 +32,7 @@ void arc_aio_cleanup()
 u32_t arc_pin_read(u8_t pin)
 {
     struct adc_seq_entry entry = {
-        .sampling_delay = 26, // anything smaller than 26 have error
+        .sampling_delay = 30,
```

Comment:
Should we really remove the comment? Seems like useful information... is it no longer true?

Reply:
@grgustaf we could leave the comment, but in reality, I don't know if anyone is going to be tweaking this value, or if they would gain any benefit by tweaking this.  All the Zephyr samples default to 30, which we had before, but I remember we lowered it to get rid of the the overhead that AIO read was causing, I remember you looked into this bug before, but I just tested the latest Zephyr, it seemed they might have fixed it.  So I think it's okay to leave it at 30 just as like other Zephyr samples.   But if you would prefer to leave the comment, I am okay with it.